<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Jian Ma :: Carnegie Mellon School of Computer Science</title>

    <!-- Bootstrap Core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Theme CSS -->
    <link href="css/freelancer.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body id="page-top" class="index">
<div id="skipnav"><a href="#maincontent">Skip to main content</a></div>

    <!-- Navigation -->
    <nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header page-scroll">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
                </button>
				<div class="navbar-brand">Ma Laboratory</div>  
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav navbar-right navbar-static-top">
                    <li class="header">
                        <a href="index.html">HOME</a>
                    </li>
                    <li class="header">
                        <a href="research.html">RESEARCH</a>
                    </li>
                    <li class="header">
                        <a href="publications.html">PAPERS</a>
                    </li>
                    <li class="header">
                        <a href="people.html">GROUP</a>
                    </li>
                     <li class="header">
                        <a href="software.html">SOFTWARE</a>
                    </li>
                     <li class="header">
                        <a href="teaching.html">TEACHING</a>
                    </li>
 
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container-fluid -->
    </nav>

    <section class="success">

	<div class="container">
	<h3>Software</h3> <br/>

        <div class="sec">More Recent Software</div>
	
	<h4><a href="http://www.nucleome.org">Nucleome Browser</a></h4>
	A multimodal, interactive data visualization platform to integrate genomic, imaging, and 3D structure models as well as external data sources to study 4D Nucleome<br/>
	<a href="https://doi.org/10.1038/s41592-022-01559-3">[Zhu et al. <i>Nat Methods</i>, 2022]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/Higashi">Higashi</a></h4>
        Multiscale and integrative single-cell Hi-C analysis.<br/>
	<a href="https://pubmed.ncbi.nlm.nih.gov/34635838/">[Zhang et al. <i>Nat Biotechnol</i>, 2021]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/DANGO">Dango</a></h4>
        Dango predicts higher-order genetic interactions.<br/>
	<a href="">[Zhang et al. <i>RECOMB</i>, 2021]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/SpiceMix">SpiceMix</a></h4>
	SpiceMix learns cell identities from spatial transcriptome data.<br/>
	<a href="">[Chidester et al. <i>RECOMB</i>, 2021]</a><br/><br/>


	<h4><a href="https://github.com/ma-compbio/SPIN">SPIN</a></h4>
        SPIN is an integrative computational method to identify genome-wide chromosome localization patterns relative to multiple nuclear bodies.<br/>
	<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02253-3">[Wang et al. <i>Genome Biol</i>, 2021]</a><br/><br/>

	<h4><a href="https://github.com/ma-compbio/MATCHA">MATCHA</a></h4>
	MATCHA enhances data quality of multi-way chromatin interaction data<br/>
	<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30147-2">[Zhang and Ma, <i>Cell Syst</i>, 2020]</a><br/><br/>

	<h4><a href="https://github.com/ma-compbio/MOCHI">MOCHI</a></h4>
	MOCHI integrates transcriptional regulation and 3D genome structure<br/>
	<a href="https://pubmed.ncbi.nlm.nih.gov/31907193/">[Tian and Zhang et al. <i>Genome Res</i>, 2020]</a><br/><br/>

	<h4><a href="https://github.com/ma-compbio/Hyper-SAGNN">Hyper-SAGNN</a></h4>
	Hyper-SAGNN is a self-attention based graph neural network architecture for hypergraphs<br/>
	<a href="https://openreview.net/forum?id=ryeHuJBtPH">[Zhang et al. <i>ICLR</i>, 2020]</a><br/><br/>

	<h4><a href="https://github.com/ma-compbio/MutSpace">MutSpace</a></h4>
	MutSpace is a cancer mutation representation learning architecture by large-scale context embedding<br/>
	<a href="https://pubmed.ncbi.nlm.nih.gov/32657413/">[Zhang et al. <i>ISMB</i>, 2020]</a><br/><br/>

	<h4><a href="https://github.com/ma-compbio/SNIPER">SNIPER</a></h4>
	SNIPER identifies 3D genome subcompartments by imputing interchromosomal Hi-C contacts<br/>
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31699985">[Xiong and Ma, <i>Nat Commun</i>, 2019]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/Phylo-HMRF">Phylo-HMRF</a></h4>
Phylo-HMRF identifies evolutionary patterns of 3D genome based on multi-species Hi-C data.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31229558">[Yang et al. <i>Cell Syst</i>, 2019]</a><br/><br/>

        <h4><a href="https://github.com/bchidest/CFNet">CFNet</a></h4>
Conic Convolution and DFT Network for classifying microscopy images.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31510662">[Chidester et al. <i>ISMB/Bioinformatics</i>, 2019]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/SPEID">SPEID</a></h4>
A deep learning model to predict enhancer-promoter interactions based on sequence-based features.
<br/>
<a href="https://link.springer.com/article/10.1007%2Fs40484-019-0154-0">[Singh et al. <i>Quantitative Biology</i>, 2019]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/Phylo-HMGP">Phylo-HMGP</a></h4>
Phylo-HMGP identifies evolutionary patterns from multi-species continuous functional genomic signals.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29936186">[Yang et al. <i>Cell Syst</i>, 2018]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/CTCF-MP">CTCF-MP</a></h4>
CTCF-MP predicts if a convergent CTCF motif pair is able to form a chromatin loop using sequence-level features.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29949986">[Zhang et al. <i>ISMB/Bioinformatics</i>, 2018]</a><br/><br/>


        <h4><a href="https://github.com/sss1/convolutional-dictionary">Convolutional Sparse Dictionary Learning</a></h4>
Implementation and experiments on covergence rates of convolutional sparse dictionary learning.
<br/>
<a href="http://proceedings.mlr.press/v84/singh18a.html">[Singh et al. <i>AISTATS</i>, 2018]</a><br/><br/>

        <h4><a href="https://github.com/bchidest/DBC">DBC</a></h4>
DBC predicts genomic markers using imaging features.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29218893">[Chidester et al. <i>Pac Symp Biocomput</i>, 2018]</a><br/><br/>


        <h4><a href="https://github.com/ma-compbio/PEP">PEP</a></h4>
PEP predicts enhancer-promoter interactions based on sequence-based features only.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28881991">[Yang et al. <i>ISMB/Bioinformatics</i>, 2017]</a><br/><br/>


        <h4><a href="https://github.com/ma-compbio/DESCHRAMBLER">DESCHRAMBLER</a></h4>
DESCHRAMBLER probabilistically reconstructs ancestral order of syntenic fragments using chromosome-scale and fragmented genome assemblies.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28630326">[Kim et al. <i>PNAS</i>, 2017]</a><br/><br/>

        <h4><a href="https://github.com/jackhou2/C3">C3</a></h4>
C3 (Cancer Correlation Clustering) identifies cancer mutation patterns from patient cohort by leveraging mutual exclusivity of mutations, patient coverage and driver network concentration principles. 
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27540270">[Hou et al. <i>Bioinformatics</i>, 2016]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/LDGM">LDGM</a></h4>
LDGM estimates differential network between two tissue types directly without inferring the network for individual tissues.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27378774">[Tian et al. <i>Nucleic Acids Res</i>, 2016]</a><br/><br/>

        <h4><a href="https://github.com/ma-compbio/Weaver">Weaver</a></h4>
Weaver quantifies allele-specific copy number alteration and allele-specific structural variation in cancer genome using whole-genome sequencing reads.
<br/>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/27453446">[Li et al. <i>Cell Syst</i>, 2016]</a><br/><br/>


	<div class="sec">Previous Software Tools</div>

            <h4><a href="http://genome.compbio.cs.cmu.edu/~jianma/software/TFBSEvo/">TFBS Evo</a></h4>
This model traces the evolution of lineage-specific transcription factor binding sites
without relying on detailed base-by-base cross-species alignments.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25144359">[Yokoyama et al. <i>PLOS Comput Biol</i>, 2014]</a><br/><br/>

            <h4><a href="http://genome.compbio.cs.cmu.edu/~jianma/software/DawnRank/">DawnRank</a></h4>
DawnRank directly prioritizes altered genes on a single patient level
which would allow us to discover potential personalized driver mutations.
<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25177370">[Hou and Ma, <i>Genome Med</i>, 2014]</a><br/><br/>


<h4><a href="http://genome.compbio.cs.cmu.edu/~jianma/software/NCIS/">NCIS</a></h4>
NCIS (network-assisted co-clustering for the identification of cancer subtypes) 
combines molecular interaction network into co-clustering.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24491042">[Liu et al. <i>BMC Bioinform</i>, 2014]</a><br/><br/>

<h4><a href="https://github.com/ma-compbio/PSAR-Align">PSAR-Align</a></h4>
PSAR-Align is a multiple sequence realignment tool that can refine a given multiple sequence alignment based on suboptimal alignments.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24222208">[Kim and Ma, <i>Bioinformatics</i>, 2013]</a><br/><br/>

<h4><a href="https://github.com/ma-compbio/RACA">RACA</a></h4>
RACA (reference-assisted chromosome assembly) is a novel algorithm 
to reliably order and orient sequence scaffolds generated by NGS and 
assemblers into longer chromosomal fragments.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23307812">[Kim et al. <i>PNAS</i>, 2013]</a><br/><br/>

<h4><a href="https://github.com/ma-compbio/TrueSight">TrueSight</a></h4>
TrueSight utilizes a machine-learning approach to precisely map RNA-seq reads to 
splice junctions, combining read mapping quality and coding potential of genomic sequences into a unified model.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23254332">[Li et al. <i>Nucleic Acids Res</i>, 2012]</a><br/><br/>

<h4><a href="http://impact.crhc.illinois.edu/Tiger/tiger.aspx">TIGER</a></h4>
TIGER is a novel de novo genome assembly framework that adapts to available computing 
resources by iteratively decomposing the assembly problem into sub-problems.
<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/23281792">[Wu et al. <i>BMC Bioinform</i>, 2012]</a><br/><br/>

<h4><a href="https://github.com/ma-compbio/PSAR">PSAR</a></h4>
PSAR is a probabilistic sampling-based alignment reliability score to assess the uncertainty 
of multiple sequence alignment. It generates suboptimal alignments based on pair-HMM.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21576232">[Kim and Ma, <i>Nucleic Acids Res</i>, 2011]</a><br/><br/>

<h4><a href="https://github.com/ma-compbio/FusionHunter">FusionHunter</a></h4>
FusionHunter reliably identifies fusion transcripts in cancer transcriptome based on paired-end RNA-seq.
It detects fusion junctions in base-level resolution.
<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/21546395">[Li et al. <i>Bioinformatics</i>, 2011]</a><br/>
<br/>

<h4><a href="http://www.bx.psu.edu/miller_lab/car/">InferCARs</a></h4>
Given a set of related genomes, InferCARs reconstructs the chromosomal architecture of their most recent
common ancestor. It also provides lineage-specific genomic breakpoint information.<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/16983148">[Ma et al. <i>Genome Res</i>, 2006]</a><br/><br/>

<h4><a href="">DupCar</a></h4>
DupCar reconstructs the ancestral genomic orders with duplications from a set of related genomes (source code available upon request).<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18774902">[Ma et al. <i>J Comput Biol</i>, 2008]</a><br/>
<br/>

<h4><a href="">Infinite Sites Model</a></h4>
The program solves the problem of recovering the history of a set of genomes that are related to an unseen common ancestor genome by operations of speciation, deletion, insertion, duplication, and rearrangement (source code available upon request).<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/18787111">[Ma et al. <i>PNAS</i>, 2008]</a><br/>
			
		</div>

    </section>

    <!-- jQuery -->
    <script src="vendor/jquery/jquery.min.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="vendor/bootstrap/js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>

    <!-- Contact Form JavaScript -->
    <script src="js/jqBootstrapValidation.js"></script>
    <script src="js/contact_me.js"></script>

    <!-- Theme JavaScript -->
    <script src="js/freelancer.min.js"></script>

</body>

</html>
